install.ps1

```diff
@@ -0,0 +1,601 @@
+<#
+.SYNOPSIS
+  Installs Rancher RKE2 to create Windows Worker Nodes.
+.DESCRIPTION
+  Run the script to install all Rancher RKE2 related needs.
+.Parameter Channel
+    Channel to use for fetching rke2 download URL.
+    Defaults to 'stable'.`
+.Parameter Method
+    The installation method to use. Currently tar or choco installation supported.
+    Default is on Windows systems is "tar".`
+.Parameter Type
+    Type of rke2 service. Only the "agent" type is supported on Windows.
+    Default is "agent".`
+.Parameter Version
+    Version of rke2 to download from github.`
+.Parameter TarPrefix
+    Installation prefix when using the tar installation method.
+    Default is C:/usr/local, unless C:/usr/local is read-only or has a dedicated mount point,
+    in which case C:/opt/rke2 is used instead.`
+.Parameter Commit
+    Commit of RKE2 to download from temporary cloud storage.
+    If set, this forces Method=tar.
+    * (for developer & QA use only)`
+.Parameter AgentImagesDir
+    Installation path for airgap images when installing from CI commit
+    Default is C:/var/lib/rancher/rke2/agent/images`
+.Parameter ArtifactPath
+    If set, the install script will use the local path for sourcing the rke2.windows-$SUFFIX and sha256sum-$ARCH.txt files
+    rather than the downloading the files from the internet.
+    Default is not set.`
+.EXAMPLE
+  Usage:
+    Invoke-WebRequest ((New-Object System.Net.WebClient).DownloadString('https://get.rke2.io/install.ps1'))        
+    ./install.ps1
+.EXAMPLE
+  Usage:
+    Invoke-WebRequest ((New-Object System.Net.WebClient).DownloadString('https://get.rke2.io/install.ps1'))        
+    ./install.ps1 -Channel Latest
+.EXAMPLE
+  Usage:
+    Invoke-WebRequest ((New-Object System.Net.WebClient).DownloadString('https://get.rke2.io/install.ps1'))        
+    ./install.ps1 -Channel Latest -Mehtod Tar
+#>
+
+    [CmdletBinding()]
+param (
+    [Parameter()]
+    [ValidateSet("latest", "stable", "testing", "v1.20", "v1.19", "v1.18")]
```

Comment:
This should come from the channel server metadata, but if you're going to hardcode it...

```suggestion
    [ValidateSet("latest", "stable", "testing", "v1.21", "v1.20", "v1.19"]
```